app/models/admin_user.rb

```diff
@@ -3,7 +3,6 @@ class AdminUser < ApplicationRecord
          password_length: 8..128
 
   validates :name, presence: true, uniqueness: true
-  validates :password, presence: true
```

Comment:
This was removed because it was messing with the edit password screen. It's not needed anyway, password is a temporary attribute, devise persists the password in the encrypted_password column and already has validations against that.